src/coveralls.jl

```diff
@@ -102,7 +110,9 @@ module Coveralls
             event_path = open(JSON.Parser.parse, ENV["GITHUB_EVENT_PATH"])
             github_pr_info = get(event_path, "pull_request", Dict())
             github_pr = get(github_pr_info, "number", "")
-            isempty(github_pr) || (data["service_pull_request"] = github_pr)
+            github_pr_string = String(strip(_stringify_string_or_int(github_pr)))
+            github_pr_string::String
+            isempty(github_pr) || (data["service_pull_request"] = github_pr_string)
```

Comment:
```suggestion
            isempty(github_pr) || (data["service_pull_request"] = strip(string(github_pr)))
```

Reply:
Oops I just pushed something that made this review "outdated", so I can't click the "commit suggestion" button, but I'll make that change manually and commit it.